react-native-libraries.json

```
@@ -15537,5 +15537,31 @@
     "android": true,
     "newArchitecture": true,
     "dev": true
+  },
+  {
+    "githubUrl": "https://github.com/react-hook-form/resolvers",
+    "npmPkg": "@hookform/resolvers",
+    "ios": true,
+    "android": true,
+    "expoGo": true,
+    "newArchitecture": true
+  },
+  {
+    "githubUrl": "https://github.com/joe-bell/cva/tree/main/packages/class-variance-authority",
+    "npmPkg": "class-variance-authority",
+    "ios": true,
+    "android": true,
+    "expoGo": true,
+    "newArchitecture": true,
+    "dev": true
```

```suggestion
    "newArchitecture": true
```

According to the docs this package should not be installed as development dependency:
* https://cva.style/docs/getting-started/installation